aws/table_aws_ec2_ami.go

```diff
@@ -103,6 +103,11 @@ func tableAwsEc2Ami(_ context.Context) *plugin.Table {
 				Type:        proto.ColumnType_STRING,
 				Default:     "self",
 			},
+			{
+				Name:        "imds_support",
```

Comment:
```suggestion
				Name:        "idms_support",
```

Is it `idms` or `imds`?

Reply:
Looking at https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/ec2@v1.73.0/types#Image, it is indeed `imds`, not `idms`, even though the description mentions `IMDS`.